### PR TITLE
v0.6.0 part 1

### PR DESCRIPTION
**Pull request by madmax983**

### CODE REVIEW

File: autumn/src/auth.rs [hunk 1]
```diff
@@ -483,6 +564,52 @@ pub struct AuthConfig {
...
+    ///
+    /// Configure in `autumn.toml`:
+    ///
+    /// ```toml
+    /// [auth.magic_link]
+    /// ttl_minutes = 15          # link lifetime; keep ≤ 15 min for a tight window
+    /// email_cooldown_secs = 60  # per-email re-mint cooldown (email-bomb throttle)
+    /// ```
+    #[serde(default)]
+    pub magic_link: MagicLinkConfig,
```
[the remark this answers is not in the document]
Comment (madmax983) [reply autumn/src/auth.rs:612]: Addressed in `6d8bcea`. `apply_auth_env_overrides_with_env` now parses both `MagicLinkConfig` fields — `AUTUMN_AUTH__MAGIC_LINK__TTL_MINUTES` and `AUTUMN_AUTH__MAGIC_LINK__EMAIL_COOLDOWN_SECS` — via the same `parse_env` helper the sibling `remember`/`password` sections use, plus a row each in the env-var reference table and unit tests (override-of-default, override-of-TOML, unset-leaves-TOML-intact, invalid-value-ignored).

---
_Generated by [Claude Code](https://claude.ai/code)_

File: autumn/src/security/submit_token.rs [hunk 2]
```diff
@@ -0,0 +1,1666 @@
...
+    let (parts, body) = req.into_parts();
+
+    let content_type = parts
+        .headers
+        .get(axum::http::header::CONTENT_TYPE)
+        .and_then(|v| v.to_str().ok())
+        .unwrap_or_default();
+    let is_urlencoded = content_type.starts_with("application/x-www-form-urlencoded");
+    let boundary = if content_type.starts_with("multipart/form-data") {
+        extract_multipart_boundary(content_type).map(str::to_owned)
```
[the remark this answers is not in the document]
Comment (madmax983) [reply autumn/src/security/submit_token.rs]: Addressed in `6030ee2`: the media type/subtype and the `boundary` parameter name are now matched case-insensitively while the boundary value stays case-sensitive (extracted from the original header), and the urlencoded branch is fixed symmetrically. Tests cover mixed-case `Multipart/Form-Data; Boundary=…` and uppercase urlencoded. A self-review of this change then caught a related bypass in the same function: the hand-rolled boundary split truncated a quoted boundary containing `;` (`boundary="x;y"`), diverging from the `multer` parser axum's `Multipart` extractor uses — fixed in `87cf3a9` by extracting the boundary with `multer::parse_boundary`.

---
_Generated by [Claude Code](https://claude.ai/code)_
